README.md

```diff
@@ -9,6 +9,7 @@ A simple SQL injection protection module that allows you to use ES6 template str
    1. [Linting](#linting)
 3. [Methods](#methods)
    1. [glue](#gluepieces-separator)
+   2. [map](#map)
```

Comment:
are you sure that this anchor link works? because the title name is different from the anchor name

Reply:
I found the problem, pushed to PR.